content/_data/changelogs/weekly.yml

```diff
@@ -6854,6 +6854,49 @@
   # pull: 4565 (PR title: Change UpdateSiteWarningsMonitor blurb punctuation)
   # pull: 4581 (PR title: Fix empty updates list)
 
+- version: 2.227
+  date: 2020-03-22
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4592
+    authors:
+    - daniel-beck
+    message: |-
+      Increase scroll speed of context menus.
+  - type: rfe
+    category: rfe
+    pull: 4589
+    authors:
+    - daniel-beck
+    message: |-
+      List plugins that failed to load on the Installed tab of the plugin manager.
+  - type: rfe
+    category: rfe
+    pull: 4584
+    authors:
+    - daniel-beck
+    message: |-
+      Highlight in the plugin manager when plugins are looking for new maintainers ("Adopt this plugin").
+  - type: rfe
+    category: developer
+    pull: 4578
+    authors:
+    - daniel-beck
+    message: |-
+      Developer: Add Javadoc for management link category definitions.
+  - type: rfe
+    category: internal
+    pull: 4594
+    issue: 61105
+    authors:
+    - olivergondza
+    message: |-
+      Internal: Permit core building using newer JDK than version 8
+
+  # pull: 4587 (PR title: Update at-since up to and including 2.226)
+  # pull: 4593 (PR title: Rework sectioning on /manage and /pluginsManager/advanced)
```

Comment:
It might be useful to have it listed as a Developer change. See https://github.com/jenkinsci/jenkins/pull/4593#pullrequestreview-377497113

Reply:
Let's not make people think that we care about themes, or we'll have to abort any UI projects right now.

Reply:
I added the sectioning rework to the changelog as 27f916c59800e1c35866550e5aff30307975cfdb.  I'm not sure I understood if @daniel-beck was inferring that we should mention the sectioning change or that we should not mention the sectioning change.  If my assumption is wrong, we can revert 27f916c59800e1c35866550e5aff30307975cfdb

Reply:
By signaling through changelog entries that only ever matter to theme authors that we care about themes, we tie our hands when it comes to the UI rework necessary to make the Jenkins UI not suck.

In the moment, these things make sense, but I doubt that's the case over a long time. 

Supporting tons of old LTS lines via update centers is similar, the current approach of supporting five lines (~ 1 year) is a hard fought compromise and sends mixed messages about what versions of Jenkins are even supported.

Another case is adding tons of changelog banners with warnings for all kinds of regressions, especially those obvious from changelog weather. I expect that once users expect this kind of service, they'll stop checking the changelog weather, to their detriment (and will end up complaining when there's regressions we didn't put up a banner for).

So I only want sustainable content that doesn't send mixed messages to users – and having changelog entries like this one does.

Reply:
See https://github.com/jenkins-infra/jenkins.io/pull/2993 for the correction @daniel-beck